server/app.js

```diff
@@ -4,22 +4,20 @@ import { join } from "path";
 import cookieParser from "cookie-parser";
 import logger from "morgan";
 import mongoose from "mongoose";
-const dotenv = require('dotenv');
 
 import indexRouter from "./routes/index";
 import pingRouter from "./routes/ping";
-import auth from "./routes/auth";
-import verifyToken from "./routes/verifyToken";
+import authRouter from "./routes/auth";
+import shopRouter from "./routes/shop";
 
-dotenv.config();
-var app = express();
+let app = express();
 
 mongoose.connect(
   process.env.DB_CONNECT,
   { useNewUrlParser: true, useUnifiedTopology: true },
-  () => console.log('connected to db!')
+  () => console.log('Mongoose has connected to the database!')
 );
-var db = mongoose.connection;
+let db = mongoose.connection;
```

Comment:
probably const right? this doesn't change?

Reply:
Good point.